packages/features/schedules/components/DateOverrideInputDialog.tsx

```diff
@@ -82,21 +82,22 @@ const DateOverrideForm = ({
 
   const form = useForm({
     values: {
-      range: value
-        ? value.map((range) => ({
-            start: new Date(
-              dayjs
-                .utc()
-                .hour(range.start.getUTCHours())
-                .minute(range.start.getUTCMinutes())
-                .second(0)
-                .format()
-            ),
-            end: new Date(
-              dayjs.utc().hour(range.end.getUTCHours()).minute(range.end.getUTCMinutes()).second(0).format()
-            ),
-          }))
-        : defaultRanges,
+      range:
+        value && value[0].start.valueOf() - value[0].end.valueOf() !== 0
```

Comment:
What do you think about replacing

```
value[0].start.valueOf() - value[0].end.valueOf() !== 0
```

with 
```
value[0].start.valueOf() !== value[0].end.valueOf()
```

I think it better reflects the logic of them being set as equal on line 131 and 132.

Reply:
sure thing